recipes/scapy/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "scapy" %}
+{% set version = "2.4.3rc2" %}
```

Comment:
we generally don't publish rcs. is there a previous version we can package instead?

Reply:
This is exceptional. There were packaging issues with 2.4.2 (missing LICENSE file declaration), leading to issues during the pip install (2.4.2 was rushed because of security concerns).
2.4.3rc2 is quite stable, and 2.4.3 will be released around next month.